README.md

```diff
@@ -389,6 +389,215 @@ In the previous examples, Docker secrets files were used to specify admin creden
+  - Create mldb_admin_password_v1.txt file to add mldb_admin_password_v1 secret for admin password using the following command:
+```
+$docker secret create mldb_admin_password_v1 mldb_admin_password_v1.txt
+```
+3. Create marklogic-multi-centos.yml using below:
+```
+version: '3.6'
+services:
+    bootstrap:
+      image: store/marklogicdb/marklogic-server:10.0-9-centos-1.0.0-ea4
+      hostname: bootstrap_3n
+      dns_search: ""
+      environment:
+        - MARKLOGIC_INIT=true
+        - MARKLOGIC_ADMIN_USERNAME_FILE=mldb_admin_username
+        - MARKLOGIC_ADMIN_PASSWORD_FILE=mldb_admin_password
+        - TZ=Europe/Prague
+      volumes:
+        - MarkLogic_3n_vol1:/var/opt/MarkLogic
+      secrets:
+          - source: mldb_admin_username_v1
+            target: mldb_admin_username
+          - source: mldb_admin_password_v1
+            target: mldb_admin_password
+      ports:
+        - 7100-7110:8000-8010
+        - 7197:7997
+      networks:
+      - external_net
+    node2:
+      image: store/marklogicdb/marklogic-server:10.0-9-centos-1.0.0-ea4
+      hostname: node2
+      dns_search: ""
+      environment:
+        - MARKLOGIC_INIT=true
+        - MARKLOGIC_ADMIN_USERNAME_FILE=mldb_admin_username
+        - MARKLOGIC_ADMIN_PASSWORD_FILE=mldb_admin_password
+        - MARKLOGIC_JOIN_CLUSTER=true
+        - TZ=Europe/Prague
+      volumes:
+        - MarkLogic_3n_vol2:/var/opt/MarkLogic
+      secrets:
+          - source: mldb_admin_username_v1
+            target: mldb_admin_username
+          - source: mldb_admin_password_v1
+            target: mldb_admin_password
+      ports:
+        - 7200-7210:8000-8010
+        - 7297:7997
+      depends_on:
+      - bootstrap
+      networks:
+      - external_net
+    node3:
+      image: store/marklogicdb/marklogic-server:10.0-9-centos-1.0.0-ea4
+      hostname: node3
+      dns_search: ""
+      environment:
+        - MARKLOGIC_INIT=true
+        - MARKLOGIC_ADMIN_USERNAME_FILE=mldb_admin_username
+        - MARKLOGIC_ADMIN_PASSWORD_FILE=mldb_admin_password
+        - MARKLOGIC_JOIN_CLUSTER=true
+        - TZ=Europe/Prague
+      volumes:
+        - MarkLogic_3n_vol3:/var/opt/MarkLogic
+      secrets:
+          - source: mldb_admin_username_v1
+            target: mldb_admin_username
+          - source: mldb_admin_password_v1
+            target: mldb_admin_password
+      ports:
+        - 7300-7310:8000-8010
+        - 7397:7997
+      depends_on:
+      - bootstrap
+      networks:
+      - external_net
+secrets:
+  mldb_admin_password_v1:
+    external: true
+  mldb_admin_username_v1:
+    external: true
+networks:
+  external_net: {}
+volumes:
+  MarkLogic_3n_vol1:
+  MarkLogic_3n_vol2:
+  MarkLogic_3n_vol3:
+```
+4. Use Docker stack to deploy the cluster using below command:
+```
+  $docker stack deploy -c marklogic-multi-centos_v1.yml mlstack
+```
+All the cluster nodes will be up and running
+Now we will rotate secrets.
+
+5. Create secrets v2 using the below commands:
```

Comment:
Create secrets v2 file using the these commands:

Reply:
missing 'file'

Reply:
Actually, we are creating docker secrets here, I will update the instruction to `create docker secrets` instead of file.